src/backend/src/services/reimbursement-requests.services.ts

```diff
@@ -692,4 +692,29 @@ export default class ReimbursementRequestService {
     if (!fileData) throw new NotFoundException('Image File', fileId);
     return fileData;
   }
+
+  /**
+   * Edits the vendor name
+   *
+   * @param name the new vendor name
+   * @param vendorId the requested vendor to be edited
+   * @param submitter the user editing the vendor name
+   * @returns the updated vendor
+   */
+  static async editVendors(name: string, vendorId: string, submitter: User) {
+    if (!isAdmin(submitter.role)) throw new AccessDeniedAdminOnlyException('only Admins can edit vendors');
```

Comment:
When I read the [ticket](https://github.com/Northeastern-Electric-Racing/FinishLine/issues/1448), it says the finance leads and heads can also edit vendors. Can we add this permission on this line?